gravitee-apim-console-webui/src/management/settings/groups/groups.component.ajs.ts

```diff
@@ -52,8 +53,13 @@ const GroupsComponentAjs: ng.IComponentOptions = {
         this.canRemoveGroup = UserService.isUserHasPermissions(['environment-group-d']);
       };
 
+      this.toggleSort = function () {
+        this.sortOrder = this.sortOrder === 'ASC' ? 'DESC' : 'ASC';
```

Comment:
```suggestion
        this.isSortingAsc = !this.isSortingAsc;
```
WDYT about changing the variable for the sort from a string to a Boolean? This simplifies the typescript code.

Reply:
We can do like this

Reply:
done